app/Actions/CreateDatabase.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace App\Actions;
+
+use App\Support\BaseAction;
+use Illuminate\Support\Facades\DB;
+
+class CreateDatabase extends BaseAction
+{
+    /**
+     * Creates the database.
+     *
+     * @return void
+     */
+    public function __invoke(): void
+    {
+        $lamboDatabase = config('lambo.database');
+
+        if ($lamboDatabase === false) {
+            return;
+        }
+
+        if ($lamboDatabase === 'mysql') {
+            $this->setDatabaseHostConfigs();
+            $this->createMySQLDatabase();
+        }
+
+        if ($lamboDatabase === 'sqlite') {
+            $this->createSqliteDatabase();
+        }
+    }
+
+    /**
+     * Set database host configuration.
+     *
+     * @return void
+     */
+    protected function setDatabaseHostConfigs(): void
+    {
+        $hostDbKeyPrefix = 'database.connections.host_database';
+
+        config()->set("{$hostDbKeyPrefix}.host", config('lambo.db_host'));
+        config()->set("{$hostDbKeyPrefix}.port", config('lambo.db_port'));
+        config()->set("{$hostDbKeyPrefix}.username", config('lambo.db_username'));
+        config()->set("{$hostDbKeyPrefix}.password", config('lambo.db_password'));
+    }
+
+    /**
+     * Creates the MySQL Database
+     *
+     * @return void
+     */
+    protected function createMySQLDatabase():void
+    {
+        try {
+            $this->console->info('Creating database...');
+
+            $connection = DB::connection('host_database');
+
+            $dbName = config('lambo-store.db_name');
+
+            $databases = $connection->select('SHOW DATABASES');
+
+            $exists = collect($databases)->filter(function ($item, $key) use ($dbName) {
+                return $item->Database === $dbName;
+            })->count();
+
+            if ($exists) {
```

Comment:
I know it's not necessary but I think I would make this named `$matchingCount` and then check whether it's `> 0` just to be more direct